src/main/java/com/crafter/springboottechiteasy/controllers/TelevisionController.java

```diff
@@ -2,56 +2,54 @@
 
 import com.crafter.springboottechiteasy.exceptions.RecordNotFoundException;
 import com.crafter.springboottechiteasy.exceptions.RequirementsNotMetException;
+import com.crafter.springboottechiteasy.models.Television;
+import com.crafter.springboottechiteasy.repositories.TelevisionRepository;
+import com.crafter.springboottechiteasy.services.TelevisionService;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @RestController
 @RequestMapping("/televisions")
 public class TelevisionController {
 
+    private TelevisionService televisionService;
```

Comment:
Wat goed dat je dit al netjes via de Service speelt. Zo hadden we dat natuurlijk al wel laten zien in de les, maar het overstijgt wel de opdracht.